SIT.Manager.Avalonia/Services/InstallerService.cs

```diff
@@ -100,15 +100,19 @@ public partial class InstallerService(IActionNotificationService actionNotificat
         /// Runs the downgrade patcher
         /// </summary>
         /// <returns>string with result</returns>
-        private async Task<string> RunPatcher() {
+        private async Task<string> RunPatcher()
+        {
             _logger.LogInformation("Starting Patcher");
             _actionNotificationService.StartActionNotification();
             _actionNotificationService.UpdateActionNotification(new ActionNotification("Running Patcher...", 100));
 
             string patcherPath = Path.Combine(_configService.Config.InstallPath, "Patcher.exe");
             if (!File.Exists(patcherPath)) {
+                patcherPath = Path.Combine(_configService.Config.InstallPath, "patcher.exe");
+                if (!File.Exists(patcherPath)) {
                 return $"Patcher.exe not found at {patcherPath}";
             }
```

Comment:
It took a lot of doc reading but there is a `Directory.GetFiles` overload that supports an `EnumerationOptions` argument which has the `MatchCasing` property you can set to false which might make this a little neater. [MS Docs](https://learn.microsoft.com/en-us/dotnet/api/system.io.directory.getfiles?view=netcore-3.1#system-io-directory-getfiles(system-string-system-string-system-io-enumerationoptions))

Reply:
Good find!

Changed it to work with that now